packages/autorest.python/autorest/codegen/models/imports.py

```diff
@@ -280,7 +283,7 @@ def add_msrest_import(
         if code_model.options["client_side_validation"]:
             if msrest_import_type == MsrestImportType.Module:
                 self.add_import(
-                    "msrest.serialization", ImportType.AZURECORE, typing_section
+                    "msrest.serialization", ImportType.SDKCORE, typing_section
```

Comment:
hmm I don't love that we are making `unbranded` a keyword-only argument, I feel like we're singling it out too much. However, it looks like `code_model` is something that we need in multiple places (see the def for this function, we require passing in `code_model`). Therefore, I think we should have `code_model` be a param to `FileImport`, we do have `code_model` pretty much everywhere. Then, the init for `FileImport` will be `def __init__(self, options, code_model)`, which is like our other objects. Then, we can do `self.code_model.unbranded` and not make it a separate thing to pass in the init. Let me do a quick PR to move code_model onto the FileImport, because it's going to be a big diff

Reply:
Agree with we shall add `code_model` as new parameter instead of `unbranded`